src/interfaces/IPRBProxyRegistry.sol

```diff
@@ -245,7 +246,7 @@ interface IPRBProxyRegistry {
     /// Requirements:
     /// - The caller must have a proxy.
     ///
-    /// @param envoy The address of the account given permission to call the target.
+    /// @param envoy The address of the account for which the permission status is set.
```

Comment:
I'd rehash the original wording like this:

> The address of the account being given or taken permission to call the target

Reply:
![image](https://github.com/PaulRBerg/prb-proxy/assets/22785348/baa4a916-77e7-4f62-8d62-6165f73ece1a)

Even though the alternatives proposed by ChatGPT aren't too good, the main point is that both the original wording and the one you've come up now is gramatically incorrect.

It's not the "adress of the account" that's being "given" or "taken" - it's the "permission" that's being "given to" or "taken from" the address.

Reply:
Good catch! Thanks for poking ChatGPT about this. Will address this in a separate PR.